includes/class-moderation.php

```diff
@@ -271,6 +271,35 @@ public static function get_site_blocks() {
 		);
 	}
 
+	/**
+	 * Check if an actor is blocked by user or site-wide.
+	 *
+	 * @param string $actor_uri Actor URI to check.
+	 * @param int    $user_id   Optional. User ID to check user blocks for. Defaults to 0 (site-wide only).
+	 * @return bool True if blocked, false otherwise.
+	 */
+	public static function actor_is_blocked( $actor_uri, $user_id = 0 ) {
```

Comment:
This sounds more like a statement than a question. What about

```suggestion
	public static function is_actor_blocked( $actor_uri, $user_id = 0 ) {
```

Reply:
Fixed in [1e6a1ee](https://github.com/Automattic/wordpress-activitypub/pull/2097/commits/1e6a1eedf1bd04bebbaa39ec8035d7850b6afc35).